tensorflow_graphics/projects/mesh_rcnn/branches/mesh_head.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2020 The TensorFlow Authors
+#
```

Comment:
branches is a vague term. 
Do you want to rename this folder stages?

And separate out code into
mesh_refinement_layer.py, and
mesh_refinement_stage.py 

this is helpful - if one wants to define a new refinement layer but re-use the stages (or vice versa)

Reply:
Maybe rename it to head? Since a layer consists of multiple stages, renaming the folder to stages would be misleading.